ios/ReactNativeHealthkit.swift

```diff
@@ -1,5 +1,6 @@
 import HealthKit
 import CoreLocation
+import WorkoutKit
```

Comment:
@robertherber just to note that WorkoutKit is only available in Xcode 15 which means that builds will fail on any xcode versions lower than this. Im not that familiar with code but is it possible to do something like `@available(iOS 17.0, *)` for imports?

Reply:
Swift is not my primary language so not quite sure on how to deal with this. I'm guessing we might need to break this out into a separate file?

ChatGPT suggested something like this, not sure if it works?
```
if #available(iOS 17.0, *) {
    import ModuleA
}
```

Reply:
@robertherber ive gone for the below which should do the trick. Ive also wrapped the code which invoke the workout function as that too will not work with a build that has been compiled with xcode < 15
```
#if canImport(WorkoutKit)
import WorkoutKit
#endif
```